app/src/index.tsx

```diff
@@ -1,9 +1,35 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { createMuiTheme } from '@material-ui/core/styles';
+import { MuiThemeProvider } from '@material-ui/core/styles';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const theme = createMuiTheme({
+    typography: {
+        fontFamily: [
+            'Work Sans',
```

Comment:
Could you paste in `public/index.html` for using Work Sans ?
<link href="https://fonts.googleapis.com/css?family=Work+Sans&display=swap" rel="stylesheet">